tests/providers/sftp/sensors/test_sftp.py

```diff
@@ -108,6 +108,17 @@ def test_naive_datetime(self, sftp_hook_mock):
         sftp_hook_mock.return_value.get_mod_time.assert_called_once_with("/path/to/file/1970-01-01.txt")
         assert not output
 
+    @patch("airflow.providers.sftp.sensors.sftp.SFTPHook")
```

Comment:
I would suggest to parameterize the newer_than rather than reapting the same code.

Reply:
Your are right, my bad. It's done now.